src/components/menuFilter/demoBasicUsage/index.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="menuFilterController as ctrl" class="md-padding" ng-cloak>
+  <div>
+    <h1 class="md-title">Pick a drink below</h1>
+    <div layout="row">
+      <md-input-container>
+        <label>Drinks</label>
+        <md-select ng-model="ctrl.selectedDrinks" multiple
+                   md-on-close="ctrl.clearSearch()"
+                   data-md-container-class="menuFilterdemoBasicUsage">
+          <md-select-header class="demo-select-header">
+            <md-menu-filter items="ctrl.drinks"
```

Comment:
@DerekLouie - I keep returning to this name `md-menu-filter` which I find VERY confusing.

* What is the intent of this directive?
* IMO, I think a better name should be used.